ankdown/ankdown.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
I was dreading the day someone would call me out on this :sweat_smile: - do you know if it will work on OS X?

Reply:
(can merge without this property, probably - I'll just need to actually test it :P)

Reply:
I work with OS X. The change made testing easier.